STATUS: CLEANER DOES NOT FIX
MODIFIED: N/A
MODIFIED FILE: N/A
CLEANER: com.alibaba.wasp.client.TestFromClientSide.testSqlExecute
CLEANER FILE: /home/alex/Repositories/wasp/src/test/java/com/alibaba/wasp/client/TestFromClientSide.java
POLLUTER: com.alibaba.wasp.client.TestAdmin.testCloseEntityGroupIfInvalidEntityGroupNameIsPassed
POLLUTER FILE: /home/alex/Repositories/wasp/src/test/java/com/alibaba/wasp/client/TestAdmin.java
ORIGINAL CLEANER SIZE: 8
NEW CLEANER SIZE: 8
ELAPSED TIME: 0.0
==========================
@@ -0,0 +0,41 @@
+ {
+     {
+         try {
+             WaspTestingUtility.adjustLogLevel();
+             TEST_UTIL.getConfiguration().setInt("wasp.client.retries.number", 3);
+             TEST_UTIL.startMiniCluster(3);
+             TableSchemaCacheReader.getInstance(TEST_UTIL.getConfiguration()).clearCache();
+             TEST_UTIL.createTable(TABLE);
+             TEST_UTIL.getWaspAdmin().disableTable(TABLE);
+             client = new FClient(TEST_UTIL.getConfiguration());
+             client.execute("create index " + INDEX_NAME + " on " + TABLE_NAME + "(column3);");
+             TEST_UTIL.getWaspAdmin().waitTableNotLocked(TABLE);
+             TEST_UTIL.getWaspAdmin().enableTable(TABLE);
+         } catch (Throwable ex) {
+         } finally {
+         }
+     }
+     {
+         try {
+         } catch (Throwable ex) {
+         } finally {
+         }
+     }
+     String INSERT = "Insert into " + TABLE_NAME + "(column1,column2,column3) values (10,10,'binlijin');";
+     Pair<String, Pair<Boolean, List<ExecuteResult>>> insertRet = client.execute(INSERT);
+     assertTrue(insertRet.getSecond().getSecond().get(0).getStatus().getOperationStatusCode() == OperationStatusCode.SUCCESS);
+     printTablesRs("testSqlExecute");
+     {
+         try {
+         } catch (Throwable ex) {
+         } finally {
+         }
+     }
+     {
+         try {
+             TEST_UTIL.shutdownMiniCluster();
+         } catch (Throwable ex) {
+         } finally {
+         }
+     }
+ }
